dist/common/scripts/scylla_setup

```diff
@@ -299,7 +299,7 @@ if __name__ == '__main__':
                     sys.exit(1)
 
     disks = args.disks
-    nic = args.nic
+    args.nic
```

Comment:
What does this do?

Reply:
Oh sorry, that was probably mistakenly removed while editing the code, reverted the line